keymanagementrequirements.md

```diff
@@ -1,79 +1,124 @@
 # Overview
+
 Key management for container signing can be broadly categorized into three general use cases:
+
 - Key Setup/Signing (Key managment and integrations with client tooling for generating signatures.)
 - Deployment configurations (Configuring trust policies and which keys to trust for artifacts.)
-- Signature Validity (Expiry, Revocation and/or Trusted Update List. Mechanism to update information on the validity of signatures.)
+- Signature Validity (Expiry, Revocation and/or Trusted Update List.
+  Mechanism to update information on the validity of signatures.)
+
+## Personas
 
-## Personas:
 - Publisher: User who builds and signs containers.
-    - Publisher Admin: In some scenarios a publisher will include a group of users (i.e. teams or enterprises). Admin users (i.e. security administrator) will be responsible for configuring roots, provide access to use or generate delegate keys, and make decisions for key revocation.
-    - Publisher Builder: In some scenarios a publisher will include a group of users (i.e. teams or enterprises). Builders (i.e. developers, build hosts) will be responsible for creating artifacts. 
-    - Publisher Signer: In some scenarios a publisher will include a group of users (i.e. teams or enterprises). Signers (i.e. developers, signing hosts) will be responsible for signing artifacts.
-- Deployer: User who deploys containers.
-    - Deployer Admin: In some scenarios a deployer will include a group of users (i.e. teams or enterprises). Admin users (i.e. security administrators) will be responsible for configuring deployment policies, including potentially a set of trusted roots or required signatures. 
-    - Deployer Operator: In some scenarios a publisher will include a group of users (i.e. teams or enterprises). Operators (i.e. developers, deployment hosts) will be responsible for pulling container images from registries, verifying their signatures, and then running them.
-- Repository Owner: In some scenarios a container may be stored in a repository that is managed by the repository owner. The repository may be public or private and could also be air-gapped.
+  - Publisher Admin:
```

Comment:
@sudo-bmitch, were you looking for a linewrap on the output? 
- Publisher Admin:  
  In some scenarios a publisher will include a group of users (i.e. teams or enterprises).
    Admin users (i.e. security administrator) will be responsible for configuring roots, provide access to use or generate delegate keys, and make decisions for key revocation.

Reply:
ping: @sudo-bmitch

Reply:
Woops, sorry @SteveLasker , missed that previous notification. I'm seeing the lines wrap in my view. Are they getting split in yours? In general I was keeping each sentence to a separate line to make PRs easier.